CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ set(shp_LIB_VERSIONINFO "4:0:0")
 set(PROJECT_VERSION
   "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 
+set(EG_DATA ${CMAKE_CURRENT_BINARY_DIR}/shape_eg_data CACHE STRING "")
+set(EG_DATA_URI http://dl.maptools.org/dl/shapelib/shape_eg_data.zip)
```

Comment:
I'm not super confident about dl.maptools.org being a long term safe archive. Couldn't we just hava a tests/data/ directory in this repository with the content of this zip ?

Reply:
Sure. This way we can also enable `test1.sh` without the hassle of downloading/extracting portably. :)